server/service/apple_mdm.go

```diff
@@ -2824,3 +2818,144 @@ func (svc *Service) getConfigAppleBMDefaultTeamID(ctx context.Context, appCfg *f
 
 	return tmID, nil
 }
+
+// scepCertRenewalThresholdDays defines the number of days before a SCEP
+// certificate must be renewed.
+const scepCertRenewalThresholdDays = 30
+
+// maxCertsRenewalPerRun specifies the maximum number of certificates to renew
+// in a single cron run.
+//
+// Assuming that the cron runs every hour, we'll enqueue 24,000 renewals per
+// day, and we have room for 24,000 * scepCertRenewalThresholdDays total
+// renewals.
+//
+// For a default of 30 days as a threshold this gives us room for a fleet of
+// 720,000 devices expiring at the same time.
+const maxCertsRenewalPerRun = 100
```

Comment:
Seems like plenty of room to start. Should we open a follow up ticket to make this calculation more dynamic based on the number of potentially expiring certs? It could also include the TODO for `GetHostCertAssociationByCertSHA`.